### DISCUSSION TITLE
Readme view has issues with markdown rendering

### DISCUSSION THREAD

**Discussion by cgsg10** [npm/feedback#1010] [Website] [answered]
New releases starting from 21/09/2023 have markdown rendering issues e.g. https://www.npmjs.com/package/@tramvai/test-mocks/v/2.149.1 compared to https://www.npmjs.com/package/@tramvai/test-mocks/v/2.149.0

**Comment by MylesBorins** [comment]
I've escalated to the eng team, ty for reporting

**Reply by lukekarrys** [reply, marked as answer]
The fix for this has now been deployed. @cgsg10 I specifically checked those pages and they are rendering correctly now.
